modules/rosa-hcp-sts-creating-a-cluster-cli.adoc

```diff
@@ -28,35 +28,47 @@ When using the {product-title} (ROSA) CLI, `rosa`, to create a cluster, you can
 //REGION="<region>"
 //----
 
-. You can create your {hcp-title} cluster with one of the following commands.
```

Comment:
@EricPonvelle I'm confused as to why you are making this change. The structure of the [current docs](https://docs.openshift.com/rosa/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html#rosa-hcp-sts-creating-a-cluster-cli_rosa-hcp-sts-creating-a-cluster-quickly) seems right to me.